includes/adminSettings.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+/* 
+ * dashboard for admin
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+require_once "../LocalSettings.php";
+require_once "Globals.php";
+
+// initial check for settings table.
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
+
+if( $sqlConn->connect_errno ) 
+{
+	echo "Error connecting database. Please check your database credentials and update.";
+	die();
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->